packages/react-menu/README.md

```diff
@@ -5,3 +5,7 @@
 These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
 
 To import React Menu components:
+
+```js
+import { MenuList, MenuItem } from '@fluentui/react-menu';
+```
```

Comment:
```suggestion
```

I would postpone adding this, until we have enough components to render a basic example

Reply:
removed